public/views/index-template.html

```diff
@@ -180,7 +180,7 @@
           <div class="preloader__logo"></div>
         </div>
       </div>
-      <div class="preloader__text">Loading Grafana</div>
+      <div class="preloader__text">Loading...</div>
       <div class="preloader__text preloader__text--fail">
         <p>
           <strong>If you're seeing this Grafana has failed to load its application files</strong>
```

Comment:
Is it possible for a customer to hit this failure case and see this text?

Reply:
If they misconfigure things they could definitely get that error. I'll update that text as well.